docs/README.md

```diff
@@ -97,7 +95,7 @@ Open the `src/App.vue` component, and replace the content between the `<template
 </template>
 ```
 
```

Comment:
I think just a bit more info here about what's going on in this code would be helpful for newcomers.